docs/how-to/use-rewards-api.md

```diff
@@ -0,0 +1,135 @@
+---
+description: Use the rewards API to view validator rewards.
+sidebar_position: 13
+---
+
+# View block rewards
+
+Use the [rewards API](https://consensys.github.io/teku/#tag/Rewards) to view the validator rewards for specific blocks.
+
+:::tip
+
+Enable the rewards API with the [`--rest-api-enabled`](../reference/cli/index.md#rest-api-enabled) command line option.
+
+A [Swagger interface is also available](../reference/rest.md#enable-the-rest-api-service). Use the [`--rest-api-docs-enabled`](../reference/cli/index.md#rest-api-docs-enabled) command line option to enable the web interface.
+
+:::
+
+## Limitations
+
+The following limitations apply:
+
+- The rewards API currently supports the `altair` fork upgrade and later.
+
+- The rewards API relies on state and block data to retrieve the reward information, meaning you'll receive limited data if the beacon node being queried is not an archive node.
+
+- Querying from a beacon node in `prune` storage mode only returns information for the the `finalized` block, and nothing before that.
```

Comment:
it's more like 'you can only query blocks from finalized to head if you are in prune mode'...

Reply:
Fixed.